src/Lykke.Service.PushNotifications.Services/SrvAppNotifications.cs

```diff
@@ -218,6 +237,40 @@ public async Task SendTextNotificationAsync(string[] notificationIds, string not
             await SendAndroidNotificationAsync(notificationsIds, gcmMessage);
         }
 
+        public async Task SendMtOrderChangedNotification(string[] notificationIds, string notificationType, string message, string orderId)
+        {
+            if (!Enum.TryParse(notificationType, out NotificationType type))
```

Comment:
Shouldn't there be any update to NotificationType?

Reply:
there are already all necessary types